htdocs/core/modules/propale/mod_propale_saphir.php

```diff
@@ -144,7 +144,7 @@ public function getNextValue($objsoc, $propal)
 		// Get entities
 		$entity = getEntity('proposalnumber', 1, $propal);
 
-		$date = $propal->date;
+		$date = isset($propal->date)?$propal->date:dol_now();
```

Comment:
since few days php min version is 7.0 https://github.com/Dolibarr/dolibarr/commit/861d8f4568cb0c996b2fa529e6bb0db54e6d7e27
I think you can write `$date = $propal->date ?? dol_now();`

Reply:
Ok it's done